process/block/sovereignChainBlock.go

```diff
@@ -1918,8 +1961,6 @@ func (scbp *sovereignChainBlockProcessor) updateState(header data.HeaderHandler,
 				log.Warn("couldn't commit trie checkpoint", "epoch", sovHdr.GetEpoch(), "error", err)
 			}
 		}()
-
-		scbp.nodesCoordinator.ShuffleOutForEpoch(header.GetEpoch())
```

Comment:
why is this deleted ?

Reply:
It was wrongfully put there in the beginning.
if we respect the metaBlock processor, this should only happen in `commitEpochStart`, which also happens for sovereign

Reply:
So it should have never been put in `updateState`